lib/addons/istio-control-plane/index.ts

```diff
@@ -0,0 +1,42 @@
+import { Construct } from "constructs";
+import merge from "ts-deepmerge";
+import { ClusterInfo } from "../../spi";
+import { HelmAddOn, HelmAddOnUserProps } from "../helm-addon";
+import { dependable } from '../../utils';
+import { ValuesSchema } from "./values";
+
+export interface IstioControlPlaneAddOnProps extends Omit<HelmAddOnUserProps, "namespace"> {
```

Comment:
Why omit namespace?

Reply:
this addon is dependent on istio-base, so I wanted to make sure that different namespaces would not be informed in each of them. But, I'm looking again, if the istio-base namespace is changed, it will fail just the same :)
I will remove the omit.